saw-core/src/SAWCore/Term/CtxTerm.hs

```diff
@@ -926,31 +541,30 @@ ctxCtorElimType ret (a_top :: Proxy (Typ a)) (d_top :: DataIdent d) c
         ctxTermsCtxHeadTail <$> fst <$> ctxVars2 params_pret prevs
```

Comment:
How do we know that this call to `ctxTermsCtxHeadTail` is safe? Worth documenting how it upholds the precondition that `prevs` is non-empty.

Reply:
Fixed in 94459a478425b5ea589e5817f60ef1473f4db7ef.